libs/core/langchain_core/language_models/base.py

```diff
@@ -74,8 +74,8 @@ class BaseLanguageModel(
     @property
     def InputType(self) -> TypeAlias:
         """Get the input type for this runnable."""
-        from langchain_core.prompts.base import StringPromptValue
         from langchain_core.prompts.chat import ChatPromptValueConcrete
+        from langchain_core.prompts.str import StringPromptValue
```

Comment:
any issues naming this `str` in case someone tries to import `from langchain_core.prompts import str`?

Reply:
mm good point. someone could `import str as lc_str` but probs best to avoid that in first place, will make it string